IdentityAccess/src/Kreta/IdentityAccess/Infrastructure/Symfony/Framework/config/config.yml

```diff
@@ -21,7 +21,7 @@ framework:
         resource: "%kernel.root_dir%/config/routing.yml"
         strict_requirements: ~
     form: ~
-    csrf_protection: ~
+    csrf_protection: false
```

Comment:
Thia may cause some security issues in the long term.  For apis with authentication is okey to disable csrf but in case of a form is submited by accessing an already authorized endpoint can be a vulnerability